tests/nhc-operator/tests/nhc_helpers.go

```
@@ -656,6 +657,148 @@ func buildNHCWithTestRemediation(name string) *unstructured.Unstructured {
+
+// buildNHCWithEscalation builds an NHC CR that uses escalatingRemediations
+// instead of a single remediationTemplate. The unhealthy condition duration
+// is set to 30s for faster test cycles.
+func buildNHCWithEscalation(name string, steps []escalationStep) *unstructured.Unstructured {
+	nhc := buildNHCForWorkers(name)
+	spec := nhcSpec(nhc)
+
+	delete(spec, "remediationTemplate")
+
+	escalations := make([]interface{}, len(steps))
+	for i, s := range steps {
```

golangci-lint (2.12.2)

[error] variable name 'i' is too short for the scope of its usage

(varnamelen)